### PR TITLE
Shorten texture test filenames.

### PR DESCRIPTION
**Pull request by kenrussell**
Shorten texture test filenames.

Some of these file names are over 255 characters which is (still, in the year 2016) causing problems on Windows machines.

Remove the element type from the test's file name; it's already present in the directory containing the test. Also remove the image-and-sub-image string. These will comfortably get all of the test names under this limit.

### CODE REVIEW

**Comment by kenrussell** [comment]
@zhenyao please review.

@xidachen FYI.

This pull request currently only contains the simple change to the generator. I'll run it and delete the old files and add the new ones before merging.

File: sdk/tests/py/tex_image_test_generator.py [hunk 1]
```diff
@@ -127,7 +127,7 @@
     
 def GenerateFilename(dimension, element_type, internal_format, format, type):
   """Generate test filename."""
-  filename = ("tex-image-and-sub-image-" + dimension + "d-with-" + element_type + "-" +
+  filename = ("tex-image-and-sub-image-" + dimension + "d-" +
```
Comment (zhenyao) [inline sdk/tests/py/tex_image_test_generator.py]: Maybe also remove "image-and-sub-image-"?

**Comment by zhenyao** [comment]
I have one suggestion, but lgtm one way or the other.  Feel free to generate the whole CL and land yourself.

File: sdk/tests/py/tex_image_test_generator.py (hunk 1 above)
Comment (kenrussell) [inline sdk/tests/py/tex_image_test_generator.py]: Good idea. Will do.

**Comment by kenrussell** [comment]
Updated the pull request with the massive renaming. Spot-checked the tests in the conformance test runner (both 1.0 and 2.0 tests). Merging now.

**Pull request merged**